packages/mwp-toaster/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "mwp-logger-plugin",
+  "version": "0.0.1",
+  "description": "Hapi logging plugin for MWP apps",
+  "engineStrict": true,
+  "engines": {
+    "node": ">=8.1.0"
+  },
+  "main": "./lib/index.js",
+  "scripts": {
+    "build": "babel src --out-dir lib --ignore test.js,test.jsx --source-maps",
+    "test": "yarn run test:unit",
+    "test:unit": "NODE_ENV=test jest --config jest.config.json",
+    "test:watch": "yarn test:unit -- --watch"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/meetup/meetup-web-platform.git"
+  },
+  "author": "Meetup",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/meetup/meetup-web-platform/issues"
+  },
+  "homepage": "https://github.com/meetup/meetup-web-platform#readme",
+  "dependencies": {},
+  "devDependencies": {
+    "babel-jest": "21.0.0",
+    "jest": "20.0.4",
+    "meetup-web-components": "^3.0.1281",
```

Comment:
when should modules be added to meetup-web-components vs being their own module? 

do we get the benefits of an independent module if meetup-web-components is a peer dependency? is there a risk of updating meetup-web-components in such a way that it passes all its tests yet breaks module?

Reply:
components in `meetup-web-components` are intended to be dumb display units that can be composed by consumers, but the `mwp-toaster` package ties the Toast components into redux state, which we don't want to set as a `meetup-web-components` dependency. I talked with MikeP, who wrote the `<Toast>` and `<Toaster>` components, about moving at least `<Toaster>` into `mwp-toaster` because its primary purpose is to be a 'controller' of dumb toasts, so it kind of straddles the line between what meetup-web-components is responsible for and what a consumer application needs - it defines display timing logic and lays out the toasts, but stops short of depending on overall application state.

I'd like to make that move in a followup ticket, however, because it involves some refactoring and making more fine-grained decisions about what goes where. As you recommended, I think I would _like_ all the Toast* related stuff to go into this repo, but that would make it more awkward to display it in storybook with the rest of the `meetup-web-components`, so I'm not sure what the best call is in the short term.

Long term, I'm expecting that `meetup-web-components` can be moved into `meetup-web-platform` as a `mwp-components` package or something, although that will take a fair amount of planning as well.

Reply:
>>Long term, I'm expecting that meetup-web-components can be moved into meetup-web-platform as a mwp-components package or something

I think I'm still unclear of the benefit of this. Can you explain?

Reply:
- fewer repos to keep track of/configure - this is the big win for simplicity because we only need one travis configuration for publishing any package, and things like eslint, flow, and storybook config can be centralized
- simpler dependency management
   - peer deps: all MWP packages increment together and are tested together, avoiding the risk that you highlighted about updating meetup-web-components in such a way that it passes all its unit tests yet breaks integration with this package
   - common dev dependencies - they can all be consolidated at the root of the repo so we don't have update them in many different places